src/constants.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
Easier to remember what 0,1,2 mean. We could alternatively pass in "error", "success", "info" when we set the message components' type - but this felt cleaner to me.